Incompatibility with nativescript-plugin-firebase

There's a version conflict between nativescript-geolocation and nativescript-plugin-firebase.

As of

https://github.com/NativeScript/nativescript-geolocation/blob/d47fc8dc45ceffa3c2a39b8c48d3099937a1e414/src/platforms/android/include.gradle#L19

nativescript-geolocation wants Google Play Services 11.4.0 but [nativescript-plugin-firebase forces me to pin to version 11.2.2](https://github.com/EddyVerbruggen/nativescript-plugin-firebase#includegradle-failed-to-apply-plugin--for-input-string-).

Is version 11.4.0 actually really required for this plugin to work or can you just pin to 11.2.2 as well? I think this is a issue for many people because nativescript-plugin-firebase is quite an important plugin... If not, what could I do about it? I really would like to be able using both of these plugins together in one app.

Here's the original error message I get after installing the nativescript-geolocation plugin and trying to build for Android.

>FAILURE: Build failed with an exception.
>
>* What went wrong:
>Execution failed for task ':processF0F1F2F3F4F5F6F7DebugGoogleServices'.
>> Please fix the version conflict either by updating the version of the google-services plugin (information about the latest version is available at https://bintray.com/android/android-tools/com.google.gms.google-services/) or updating the version of com.google.android.gms to 11.4.0.

>>>

--- a/README.md
+++ b/README.md
@@ -96,6 +96,23 @@ geolocation.getCurrentLocation({ desiredAccuracy: Accuracy.high, maximumAge: 500
 | isEnabled | Promise\<boolean\>| Resolves `true` or `false` based on the location services availability.  |
 | distance(loc1: Location, loc2: Location) | number | Calculate the distance between two locations. Returns the distance in meters. |
 
+## Known Issues
+
+### Version Conflicts – Google Play Services
+
+If you have installed multiple plugins that use the Google Play Services you might run into version conflicts. 
+In order to fix this you might pin the version number in your `app/App_Resources/Android/app.gradle` file:
+
+```gradle
+android {  
+  // other stuff here
+
+  project.ext {
+    googlePlayServicesVersion = "11.2.+"
+  }
+}
+```
+
 ## License
 
 Apache License Version 2.0, January 2004

--- a/src/platforms/android/include.gradle
+++ b/src/platforms/android/include.gradle
@@ -16,5 +16,6 @@ android {
 }
 
 dependencies {
-	compile "com.google.android.gms:play-services-location:11.4.0"
+	def googlePlayServicesVersion = project.hasProperty('googlePlayServicesVersion') ? project.googlePlayServicesVersion : "11.4.0"
+	compile "com.google.android.gms:play-services-location:$googlePlayServicesVersion"
 }

